.bazelrc

```diff
@@ -59,6 +60,7 @@ build:libc++ --action_env=CC
 build:libc++ --action_env=CXX
 build:libc++ --action_env=CXXFLAGS=-stdlib=libc++
 build:libc++ --action_env=PATH
+build:libc++ --host_linkopt=-fuse-ld=lld
```

Comment:
This makes me a little uneasy. Do you know _why_ we need this as `host_linkopt` vs `linkopt`?

Reply:
because `-stdlib=libc++` applies to host configuration too, `ld.gold` is unable to link libc++.a/libc++abi.a for grpc_cpp_plugin, etc. so this is needed.